code/modules/reagents/chemistry_properties/prop_neutral.dm

```diff
@@ -234,26 +234,26 @@
 	M.apply_damage(0.5 * potency * delta_time, BRAIN)
 	M.apply_effect(20, PARALYZE)
 
-/datum/chem_property/neutral/relaxing
-	name = PROPERTY_RELAXING
+/datum/chem_property/neutral/antispasmodic
+	name = PROPERTY_ANTISPASMODIC
 	code = "RLX"
```

Comment:
should probably change the code names for all the property names you modified? 

and preferably have the changelog show the names before/after for research players.